openmc/universe.py

```diff
@@ -282,12 +296,12 @@ def get_offset(self, path, filter_offset):
 
         # If the Cell is filled by a Universe
         elif self._type == 'fill':
-            offset = self._offsets[filter_offset-1]
-            offset += self._fill.get_offset(path, filter_offset)
+            offset = self._offsets[distribcell_ind-1]
```

Comment:
Use the `offsets` and `fill` property getters (if they exist, which I think they do)